patches_for_WebRTC_org/m80/readme.md

```diff
@@ -2,15 +2,15 @@
 
 This document will walk you through the steps required for getting WebRTC code base, patching it and build it for UWP.
 
-These patches are being contributed back. Some of these patches were already merged into their original repos, but didn't rolled over WebRTC yet. Contribute changes back take time. We want you to be able to build Windows apps with real time communications as soon as possible.
+These patches are being contributed back. Some of these patches were already merged into their original repos, but didn't rolled over WebRTC yet. Contributing changes back take time. We want you to be able to build Windows apps with real time communications as soon as possible.
 
 Building WebRTC from scratch is extremely powerful, but it is a lot of work. If you are looking for shipping a limited set of codecs or greater control of the binary size, building from scratch is the way to go. However, we believe that most developers would be ok with a more off of shelf solution.
 
 We're also investigating the feasibility of having NuGet packages of a patched WebRTC. Building against WebRTC requires a considerable amount of header files from multiple different repos. Besides that, there is the problem of the sheer size of the binaries produced. Stay tuned because this repo will be updated if/when a NuGet package is available.
 
 # Prerequisites
 
-To build WebRTC rapidly requires a biffy machine. Make sure your build rig is a **64-bit Intel** based machine running **Windows 10 build 18362 or more recent**. It builds on machines with 8GB of RAM, but at least **16GB of RAM** is recommended. It also requires a seriously amount of disk space. Make sure you have at least **100GB available**. Finally, make sure you **SSD** drive is formatted with **NTFS**.

```

Comment:
@LoadLibrary 100GB sound like a lot. My repo is 11.4 GB + 1.5GB per build. Maybe 15GB is more reasonable?

Reply:
I've got this number from the official Chrome build for windows instructions ([https://chromium.googlesource.com/chromium/src/+/master/docs/windows_build_instructions.md](https://chromium.googlesource.com/chromium/src/+/master/docs/windows_build_instructions.md)). I've also took a look at the size it takes and, you're right, 15GB looks much more reasonable. I'll update in the next review. Thanks.

---
In reply to: [397835292](https://github.com/microsoft/winrtc/pull/6#discussion_r397835292) [](ancestors = 397835292)